frontend/src/shared/error/errors.js

```diff
@@ -81,15 +77,15 @@ export default class Errors {
 
     if (selectErrorCode(error) === 542) {
       Message.error(
-        'An error has occurred setting up the integration, please reach out to us via chat, or via email (help@crowd.dev)',
-        { duration: 0 },
+        "An error has occurred setting up the integration, please reach out to us via chat.",
+        { duration: 0 }
       );
       return;
     }
 
     Message.error(
-      'Please try again. If the problem remains, reach out to us.',
-      { title: 'Oops, something went wrong' },
```

Comment:
Our linter is forcing to use single quotes so please run `npm run lint:fix`

Reply:
done